File: rust/src/models/docusign_config.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
pub struct DocusignConfig {
    #[serde(skip_serializing_if = "Option::is_none")]
    pub secret: Option<String>,
}

impl DocusignConfig {
    pub fn new() -> Self {
        Self { secret: None }
    }
}

impl Default for DocusignConfig {
    fn default() -> Self {
        Self::new()
    }
}

File: javascript/src/models/docusignConfig.ts
// this file is @generated

export interface DocusignConfig {
  secret?: string | null;
}

export const DocusignConfigSerializer = {
  _fromJsonObject(object: any): DocusignConfig {
    return {
      secret: object["secret"],
    };
  },

  _toJsonObject(self: DocusignConfig): any {
    return {
      secret: self.secret,
    };
  },
};
